### DISCUSSION TITLE
What is your favorite niconico douga?

### DISCUSSION THREAD

**Discussion by tktkbohshi** [NAIST-SE/AttractingContributors#53] [General]
Please fill my list. (and let's create a wonderful play list!)  
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Comment by nosfill** [comment]
added! RTA is good to watch.

**Reply by tktkbohshi** [reply]
Thanks!

**Comment by satokibi44** [comment]
added! 
Please send PR to my [repo](https://github.com/satokibi44/AttractingContributors-Anime)

**Reply by tktkbohshi** [reply]
i will PR tomorrow. Thanks!
